lua/entities/gmod_wire_hologram.lua

```diff
@@ -120,14 +120,20 @@ if CLIENT then
 		end
 
 		self:SetupClipping()
-
+		
+		render.CullMode(self:GetNWBool("invert_model", 0))
+		
 		if self:GetNWBool("disable_shading") then
 			render.SuppressEngineLighting(true)
 			self:DrawModel()
 			render.SuppressEngineLighting(false)
 		else
 			self:DrawModel()
 		end
+		
+		if self:GetNWBool("invert_model") then
```

Comment:
This `if` is pointless right now assuming `GetNWBool` would return a number (which is always truthy) from when you `SetNWBool`.